packages/amplify-category-api/src/provider-utils/awscloudformation/api-input-manager/appsync-api-input-state.ts

```diff
@@ -0,0 +1,42 @@
+import { AmplifyCategories, AmplifySupportedService, CLIInputSchemaValidator, JSONUtilities, pathManager } from 'amplify-cli-core';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { AppSyncCLIInputs } from '../service-walkthrough-types/appsync-user-input-types';
+
+export class AppsyncApiInputState {
```

Comment:
maybe too late at this point but would recommend to rename to `AppSyncApiInputState`